geode-core/src/main/java/org/apache/geode/internal/Version.java

```diff
@@ -246,8 +246,8 @@ private Version(String product, String name, byte major, byte minor, byte releas
     this.release = release;
     this.patch = patch;
     this.ordinal = ordinal;
-    this.methodSuffix = this.productName + "_" + this.majorVersion + "_" + this.minorVersion + "_"
-        + this.release + "_" + this.patch;
+    this.methodSuffix =
+        this.majorVersion + "_" + this.minorVersion + "_" + this.release + "_" + this.patch;
```

Comment:
Would this have any impact on values serialized into a disk store?

Reply:
Version isn't serializable.  Oddly some of its instance variables are marked _transient_